tvb_documentation/tutorials/tutorial_0_GettingStarted.rst

```diff
@@ -4,189 +4,220 @@
 Getting Started
 ===============
 
-First things first. We need you to upload the projects from the `TVB sharing
-area <http://www.thevirtualbrain.org/tvb/zwei/client-area/public>`_. This might
-take some time, so while we continue with the program you can start importing
-the projects into your copy of TVB.
-
-============================    ========
- **Name**                       **Size** 
-----------------------------    --------
- LinkandShare_a                  11 MB
- LinkandShare_b                  62 KB
- BuildingYourOWnBNM              1.3 GB
- Resting State Netowrks          1 GB 
- ModelingEpilepsy                411 MB 
-============================    ========   
-
-------------------------
+First things first. We are going to use *tvb-data* which is present within the distribution
+(at TVB_Distribution/tvb_data/Lib/site-packages/tvb-data)
+or it can be downloaded from `here <https://zenodo.org/record/4263723#.YL9x4jaA7t0>`_.
+Other than that, please follow
+`this link <https://www.thevirtualbrain.org/tvb/zwei/client-area/public?_ga=2.222814555.1504534223.1522695590-1355432300.1522695590#>`_
+to download the *ModelingEpilepsy.zip (3)* project.
```

Comment:
Could we keep directly ModelingEpilepsy.zip on ladeluke and rename the original as ModelingEpilepsy_old.zip?